packages/waku/src/router/fs-router.ts

```diff
@@ -77,6 +77,19 @@ export function fsRouter(
             render: 'static',
             ...config,
           });
+        } else if (
+          process.env.npm_lifecycle_script?.includes('static') &&
+          config?.render === 'dynamic'
+        ) {
+          console.warn(
+            `[WARN] getConfig for path: '${path}' cannot be dynamic for static deploys. Switching to static.`,
```

Comment:
I would just warn it and not automatically switch to static.

Reply:
will make this change when resolving the other comment, thanks!